### PR TITLE
M2-6298: Add UAT entitlements files

### PR DESCRIPTION
**Pull request by sultanofcardio**
### 📝 Description

🔗 [Jira Ticket M2-6298](https://mindlogger.atlassian.net/browse/M2-6298)

This PR adds the missing UAT entitlements files so that the correct associated domain may be used in the UAT build. The app was previously using the staging entitlements files for the UAT build

### CODE REVIEW

File: ios/MindloggerMobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -102,7 +102,7 @@
 		4E9EC54929BA279C002F69A5 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4E9EC54629BA279C002F69A5 /* MaterialIcons.ttf */; };
 		4E9EC54A29BA279C002F69A5 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4E9EC54629BA279C002F69A5 /* MaterialIcons.ttf */; };
 		4EBCD5B22971514E00D3C04C /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4EBCD5B12971514E00D3C04C /* Foundation.ttf */; };
-		51A6E1C8BEDF3FA779731DCF /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		51A6E1C8BEDF3FA779731DCF /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```
Comment (sultanofcardio) [inline ios/MindloggerMobile.xcodeproj/project.pbxproj:105]: I'm not sure why this and [this](https://github.com/ChildMindInstitute/mindlogger-app-refactor/blob/a94b95d14801dcc90ebf841b16dfd2cf9fdc1616/ios/MindloggerMobile.xcodeproj/project.pbxproj#L341) changed to null. @BamMironov do you have any idea?

Comment (BamMironov) [reply ios/MindloggerMobile.xcodeproj/project.pbxproj:105]: Hmm, that is not the first time I see this.
Do you use [bundler](https://tingyishih.medium.com/why-bundle-exec-whats-the-difference-ae7ba488a324)? 
If not, please give it a try/
I usually run it with "yarn pods" command.

Comment (sultanofcardio) [reply ios/MindloggerMobile.xcodeproj/project.pbxproj]: Yup, that did it. Thanks 🙏

**Comment by sultanofcardio** [comment]
As discussed in Slack, this change is meant for the subsequent release so we'll change the PR base when the branch is created (presumably `release/1.3.2`)

**Comment by BamMironov** [comment]
I think we could merge it to `dev` now and then (a bit later) I'll create a new branch (`release/1.3.2`) from `dev`.

**Review by BamMironov — APPROVED** [review]
Everything looks good!

**Pull request merged**